src/EnergyPlus/HVACFan.cc

```diff
@@ -305,15 +308,52 @@ namespace HVACFan {
             OutputReportPredefined::PreDefTableEntry(OutputReportPredefined::pdchFanPwrPerFlow, name, designElecPower / designAirVolFlowRate);
         }
         OutputReportPredefined::PreDefTableEntry(OutputReportPredefined::pdchFanMotorIn, name, m_motorInAirFrac);
+        OutputReportPredefined::PreDefTableEntry(OutputReportPredefined::pdchFanEnergyIndex, name, m_designPointFEI);
+
         OutputReportPredefined::PreDefTableEntry(OutputReportPredefined::pdchFanEndUse, name, m_endUseSubcategoryName);
 
         m_objSizingFlag = false;
     }
 
+    void FanSystem::report_fei()
```

Comment:
This whole function is repeated?? Why? But with subtle differences of the density calculations? 

Nevermind, this isn't OK.  And the unit test below only covers one of these functions.

Please make a single function that is callable from any fan.  You can leave this method in the FanSystem object, but don't repeat the logic and mathematics.  Have this method call out to your new single function to calculate FEI and return.